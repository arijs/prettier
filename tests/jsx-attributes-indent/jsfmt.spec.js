run_spec(__dirname, { jsxAttributesIndent: true, useTabs: false }, ["typescript"]);
run_spec(__dirname, { jsxAttributesIndent: false, useTabs: false }, ["typescript"]);
run_spec(__dirname, { jsxAttributesIndent: true, useTabs: true }, ["typescript"]);
run_spec(__dirname, { jsxAttributesIndent: false, useTabs: true }, ["typescript"]);
