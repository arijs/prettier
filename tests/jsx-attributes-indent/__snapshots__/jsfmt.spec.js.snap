// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test.js 1`] = `
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  bigObject={{
    test: value,
    test2: otherValue,
  }}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<SomeHighlyConfiguredComponent
 onEnter={this.onEnter}
 onLeave={this.onLeave}
 onChange={this.onChange}
 initialValue={this.state.initialValue}
 bigObject={{
  test: value,
  test2: otherValue
 }}
 ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>;

`;

exports[`test.js 2`] = `
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  bigObject={{
    test: value,
    test2: otherValue,
  }}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  bigObject={{
    test: value,
    test2: otherValue
  }}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>;

`;

exports[`test.js 3`] = `
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  bigObject={{
    test: value,
    test2: otherValue,
  }}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<SomeHighlyConfiguredComponent
 onEnter={this.onEnter}
 onLeave={this.onLeave}
 onChange={this.onChange}
 initialValue={this.state.initialValue}
 bigObject={{
	test: value,
	test2: otherValue
 }}
 ignoreStuff={true}
>
	<div>and the children go here</div>
	<div>and here too</div>
</SomeHighlyConfiguredComponent>;

`;

exports[`test.js 4`] = `
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  bigObject={{
    test: value,
    test2: otherValue,
  }}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<SomeHighlyConfiguredComponent
	onEnter={this.onEnter}
	onLeave={this.onLeave}
	onChange={this.onChange}
	initialValue={this.state.initialValue}
	bigObject={{
		test: value,
		test2: otherValue
	}}
	ignoreStuff={true}
>
	<div>and the children go here</div>
	<div>and here too</div>
</SomeHighlyConfiguredComponent>;

`;
